app/controllers/api/v2/config_templates_controller.rb

```diff
@@ -96,13 +98,29 @@ def build_pxe_default
 
       def clone
         @config_template = @config_template.clone
-        load_vars_from_config_template
+        load_vars_from_template
         @config_template.name = params[:config_template][:name]
         process_response @config_template.save
       end
 
+      def resource_name
+        'config_template'
+      end
+
       private
 
+      def type_name_singular
+        @type_name_singular ||= resource_name
+      end
+
+      def deprecated
+        ::ActiveSupport::Deprecation.warn('Config templates were renamed to provisioning templates')
```

Comment:
I would suggest mentioning the paths it the warning message, so that when I do this mistake, I know, what path to use 'The resources /config_templates were moved to /provisioning_templates. Use the new path instead.'